src/components/AnalysisPage/ThicknessAnalysisBox/ThickAnalysisBox.jsx

```diff
@@ -63,6 +67,19 @@ const ThickAnalysisBox = ({ partId, thickData }) => {
           />
         </Col>
       </Row>
+      {partsThresholdHistory && (
+        <div style={{ marginTop: ".5em" }}>
+          <Row>
+            <Segmented
+              options={partsThresholdHistory.map((threshold) => ({
+                label: `${threshold}`,
+                value: threshold,
+                key: `${threshold}`,
+              }))}
+            />
+          </Row>
+        </div>
```

Comment:
![highlight-one-thickness](https://github.com/rknizzle/propermesh-view/assets/40612461/efed5b0f-5f3f-4ca0-88d6-368f35bc0054)

I like how the box gets highlighted when it gets moused over so it looks clickable. I eventually want to do this same visual to the view parts rows I think so that the row looks clickable